gov-notify-stub/src/MockGovNotifyHandler.ts

```diff
@@ -25,41 +24,32 @@ class MockGovNotifyHandler implements LambdaInterface {
 
 	@metrics.logMetrics({ throwOnEmptyMetrics: false, captureColdStartMetric: true })
 	async handler(event: APIGatewayProxyEvent, context: any): Promise<APIGatewayProxyResult> {
-		 switch (event.resource) {
-			 case ResourcesEnum.GOV_NOTIFY_SEND_EMAIL:
-				 if (event.httpMethod === "POST") {
-					 try {
-						 logger.info("Event received: GOVNotify SendEmail", { event });
-						 logger.info("ABOUT TO PARSE JSON");
-						 const payload = event.body;
-						 let payloadParsed;
-
-						 if (payload) {
-							 logger.info("Event body", { payload });
-							 if (event.isBase64Encoded) {
-								 payloadParsed = JSON.parse(Buffer.from(payload, "base64").toString("binary"));
-							 } else {
-								 payloadParsed = JSON.parse(payload);
-							 }
-
-							 logger.info("PARSED JSON", { payloadParsed });
-							 logger.info("PARSED EMAIL", payloadParsed.email_address);
-							 logger.info("FINISHED PARSING, awaiting return");
-							 return await GovNotifyRequestProcessor.getInstance(logger, metrics).mockSendEmail(payloadParsed.email_address);
-						 }
-
-					 } catch (err: any) {
-						 logger.error({ message: "An error has occurred.", err });
-						 if (err instanceof AppError) {
-							 return new Response(err.statusCode, err.message);
-						 }
-						 return new Response(HttpCodesEnum.SERVER_ERROR, "An error has occurred");
-					 }
-				 }
-				 break;
-					
-			 default:
-			 	throw new AppError(`Requested resource does not exist: ${event.resource}`, HttpCodesEnum.NOT_FOUND);
+		try {
+			logger.info("Event received: GOVNotify SendEmail", { event });
+			logger.info("ABOUT TO PARSE JSON");
+			const payload = event.body;
+			let payloadParsed;
+
+			if (payload) {
+				logger.info("Event body", { payload });
+				if (event.isBase64Encoded) {
+					payloadParsed = JSON.parse(Buffer.from(payload, "base64").toString("binary"));
+				} else {
+					payloadParsed = JSON.parse(payload);
+				}
+
+				logger.info("PARSED JSON", { payloadParsed });
+				logger.info("PARSED EMAIL", payloadParsed.email_address);
+				logger.info("FINISHED PARSING, awaiting return");
+				return await GovNotifyRequestProcessor.getInstance(logger, metrics).mockSendEmail(payloadParsed.email_address);
+			}
+
+		} catch (err: any) {
+			logger.error({ message: "An error has occurred.", err });
+			if (err instanceof AppError) {
```

Comment:
[nit] you had a really nice ternary in your other PR, want to do that here too? Think it was something like -

```
return err instanceof AppError ? 
  new Response(err.statusCode, err.message)
  : new Response(HttpCodesEnum.SERVER_ERROR, "An error has occurred")

Reply:
Done